src/cli_plugin/cli.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import pkg from '../utils/packageJson';
+import Command from '../cli/Command';
+import listCommand from './list';
+import installCommand from './install';
+import removeCommand from './remove';
+
+let argv = process.env.kbnWorkerArgv ? JSON.parse(process.env.kbnWorkerArgv) : process.argv.slice();
+let program = new Command('bin/kibana-plugin');
+
+program
+.version(pkg.version)
+.description(
+  'Kibana is an open source (Apache Licensed), browser ' +
+  'based analytics and search dashboard for Elasticsearch.'
```

Comment:
Should this description be more related to the plugin installer tool instead of Kibana as a whole?